app/actions/definitions/comments.tsx

```diff
@@ -55,7 +55,10 @@ export const resolveCommentFactory = ({
 
       history.replace({
         ...history.location,
-        state: null,
+        state: {
+          ...(history.location.state as Record<string, unknown>),
```

Comment:
I'm a bit worried about spreading the entire state like this, perhaps we should be picking just the `sidebarContext`? I don't want to have other random state following between navigations 🤔

Reply:
Thought of being extra safe and send everythin, but you're right - there could be unnecessary state we may not be interested in that lead to bugs.